package.json

```diff
@@ -3,13 +3,15 @@
   , "description": "NodeJS client for the Medium app"
   , "version": "0.0.1"
   , "homepage": "https://github.com/medium/medium-sdk-nodejs"
-  , "authors": [
-    "Jamie Talbot <jamie@jamietalbot.com> (https://github.com/majelbstoat)"
-  ]
+  , "author": "Jamie Talbot <jamie@jamietalbot.com> (https://github.com/majelbstoat)"
```

Comment:
'author' takes a single name, there's no plural in the package.json reference. (https://docs.npmjs.com/files/package.json#people-fields-author-contributors)